staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter.go

```diff
@@ -388,12 +388,13 @@ func interfaceFromUnstructured(sv, dv reflect.Value) error {
 	return nil
 }
 
-func (c *converterImpl) ToUnstructured(obj interface{}, u *map[string]interface{}) error {
+func (c *converterImpl) ToUnstructured(obj interface{}) (map[string]interface{}, error) {
```

Comment:
I don't understand the question, sorry. Basically the only change I made is I pulled in the map allocation into the function and made it return that map. No functional change. This function "api" change makes it impossible to misuse the function though.